django/db/models/options.py

```diff
@@ -218,6 +218,32 @@ def _format_names_with_class(self, cls, objs):
             new_objs.append(obj)
         return new_objs
 
+    def _get_default_pk_class(self):
+        pk_class_path = getattr(
+            self.app_config,
+            'default_auto_field',
+            settings.DEFAULT_AUTO_FIELD,
+        )
+        try:
+            pk_class = import_string(pk_class_path)
+        except ImportError as e:
+            msg = f"The module '{pk_class_path}' could not be imported. "
+            if settings.DEFAULT_AUTO_FIELD != pk_class_path:
+                app_config_class = type(self.app_config)
+                msg += (
+                    f"Check your '{app_config_class.__module__}."
+                    f"{app_config_class.__qualname__}.default_auto_field' "
+                    f"attribute."
+                )
+            else:
+                msg += 'Check your DEFAULT_AUTO_FIELD setting.'
+            raise ImproperlyConfigured(msg) from e
+        if not issubclass(pk_class, AutoField):
+            raise ValueError(
+                f"Primary key '{pk_class_path}' must subclass AutoField."
```

Comment:
Do you think it would be helpful to include the app config path in this message, similar to how you do it above on line 234?

Reply:
Done.